airflow/providers/apache/livy/hooks/livy.py

```diff
@@ -155,7 +155,7 @@ def post_batch(self, *args: Any, **kwargs: Any) -> int:
         """
         batch_submit_body = json.dumps(self.build_post_batch_body(*args, **kwargs))
 
-        if self.base_url is None:
+        if self.base_url in (None, ""):
```

Comment:
```suggestion
        if self.base_url:
```

Reply:
then it will also covers `self.base_url` = `[]` and `{}`. is that acceptable? I want to avoid extending beyond the original commit’s scope.

Reply:
any further feedback @potiuk ?

Reply:
It's just **simpler check** - and more pythonic. No harm

Reply:
OK. and since `base_url` is hinted as a `str` in the parent`init`, we should probability be fine. will push a commit.